lib/bunyan.js

```diff
@@ -155,7 +155,7 @@ if (!format) {
             switch (x) {
                 case '%s': return String(args[i++]);
                 case '%d': return Number(args[i++]);
-                case '%j': return JSON.stringify(args[i++], safeCycles());
+                case '%j': return safeStringify(args[i++]);
```

Comment:
Perhaps have "Json" in that function name (ignore what I said earlier about maniacally avoiding "safeJsonStringify" as a name).